org_fedora_oscap/service/kickstart.py

```diff
@@ -249,5 +236,7 @@ class OSCAPKickstartSpecification(KickstartSpecification):
     """The kickstart specification of the OSCAP service."""
 
     addons = {
-        "org_fedora_oscap": OSCAPKickstartData
+        "org_fedora_oscap": OSCAPKickstartData,
+        "org_centos_oscap": OSCAPKickstartData,
+        "com_redhat_oscap": OSCAPKickstartData,
```

Comment:
These variants can be tested in unit tests. There is also support for checking errors and warnings.